sos/report/plugins/libvirt.py

```diff
@@ -6,16 +6,16 @@
 #
 # See the LICENSE file in the source distribution for further information.
 
-from sos.report.plugins import Plugin, IndependentPlugin
+from sos.report.plugins import Plugin, DebianPlugin, RedHatPlugin, UbuntuPlugin
 import glob
 
 
-class Libvirt(Plugin, IndependentPlugin):
+class Libvirt(Plugin, DebianPlugin, RedHatPlugin, UbuntuPlugin):
```

Comment:
You don't need `RedHatPlugin` here, as the functions in this class will get triggered via `RedHatLibvirt` as the `RedHatPlugin` is being used there

Reply:
ack, i took it as example another [plugin](https://github.com/sosreport/sos/blob/main/sos/report/plugins/samba.py), but I guess you're right.